packages/gitbook/src/app/(space)/(core)/~gitbook/ogimage/[pageId]/route.tsx

```diff
@@ -1,53 +1,210 @@
+import { CustomizationHeaderPreset } from '@gitbook/api';
 import { redirect } from 'next/navigation';
 import { ImageResponse } from 'next/og';
-import { NextRequest } from 'next/server';
+import { ImageResponseOptions, NextRequest } from 'next/server';
+import colorContrast from 'postcss-color-contrast/js';
 import React from 'react';
 
+import { absoluteHref } from '@/lib/links';
+import { tcls } from '@/lib/tailwind';
 import { getContentTitle } from '@/lib/utils';
 
 import { PageIdParams, fetchPageData } from '../../../../fetch';
 
 export const runtime = 'edge';
 
+// TODO: Support all fonts available in GitBook
+// Right now this is impossible since next/font/google does not expose the cached font file
+// Another option would be to use the Satori prop `loadAdditionalAsset` [example](https://github.com/vercel/satori/blob/main/playground/pages/index.tsx),
+// but this prop isn't (yet) exposed through `ImageResponse`.
+const interRegular = fetch(
+    new URL('../../../../../../fonts/Inter/Inter-Regular.ttf', import.meta.url),
+).then((res) => res.arrayBuffer());
+const interBold = fetch(
+    new URL('../../../../../../fonts/Inter/Inter-Bold.ttf', import.meta.url),
```

Comment:
Does it actually work? I'm surprised, it should be possible to import them instead.

Also should this fetch be moved in the `GET` function instead

Reply:
Ah yes https://vercel.com/guides/using-custom-font ok 😅